carrental/Rental/models.py

```diff
@@ -35,30 +38,22 @@ class UserProfile(BaseModel):
     """UserProfile model"""
 
     user = models.OneToOneField(User, on_delete=models.CASCADE)
-    phone = models.CharField(max_length=64, null=False)
-    address = models.CharField(max_length=64, null=False)
+    date_of_birth = models.DateField(blank=True, null=True)
+    photo = models.ImageField(upload_to='users/%Y/%m/%d',blank=True)
 
     def __str__(self):
-        """Return name of UserProfile"""
-        return f'{self.user} {self.address} {self.phone}'
+        return {self.user.username}
 
 
-class Rental(BaseModel):
-    """Rental model"""
 
+class Rental(models.Model):
     car = models.ForeignKey(Car, on_delete=models.CASCADE, related_name='rentals', null=False)
     user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='rentals', null=False)
     start_date = models.DateField(null=False)
     end_date = models.DateField(null=False)
-    price = models.DecimalField(max_digits=6, decimal_places=2, blank=True)
-
-    def __str__(self):
-        """Return name of rental"""
-        return f"{self.car} {self.user} {self.start_date} {self.end_date}"
-
-    def save(self, *args, **kwargs):
-        if not self.pk:
-            self.price = self.car.price * (self.end_date - self.start_date).days
+    price = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)  # change to null=True
```

Comment:
nie piszmy takich rzeczy w komentarzu, to nie miejsce na informacje o historiach zmian